tapdb/sqlc/queries/assets.sql

```diff
@@ -398,7 +400,14 @@ JOIN internal_keys
 JOIN managed_utxos utxos
     ON assets.anchor_utxo_id = utxos.utxo_id AND
       (utxos.outpoint = sqlc.narg('anchor_point') OR
-       sqlc.narg('anchor_point') IS NULL)
+       sqlc.narg('anchor_point') IS NULL) AND
+       CASE
+           WHEN sqlc.narg('leased') = true THEN
+               (utxos.lease_owner IS NOT NULL AND utxos.lease_expiry > @now)
+           WHEN sqlc.narg('leased') = false THEN
+               (utxos.lease_owner IS NULL OR utxos.lease_expiry <= @now)
```

Comment:
Perhaps we should add `OR utxos.lease_expiry IS NULL` to this `WHEN` clause.

Reply:
Good catch, added.